### DISCUSSION TITLE
About library name suggestion (#55)

### DISCUSSION THREAD

**Discussion by gftea** [gftea/amqprs#56] [General]
suggestion from #55  that do not use `amqp` in lib name.

Would like to hear more opinions or discussion.

**Comment by cdbennett** [comment]
I see the point in that issue, that AMQP is really two different things AMQP 1.x, and the RabbitMQ protocol (0.9.1) is its own ecosystem.  If amqprs is going to be renamed, might as well do it soon so there are fewer apps depending on it that need to change.

It would be nice if the name was

- short enough to be easy to type
- had some connection to RabbitMQ

The name could try to communicate what makes this crate unique and what it's specific goals are, but (1) that's a lot to try to do in a concise name, and (2) those specific goals could change over time.

After thinking about it, my vote is to keep the existing name, just make sure the crate description communicates it's for RabbitMQ and AMQP 0-9-1.  Which it already does, here is what you see if you search for it on crates.io:
> amqprs v1.0.9
> AMQP 0-9-1 client implementation for RabbitMQ
